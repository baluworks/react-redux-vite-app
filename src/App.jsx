import "./App.css";

function App() {
  return (
    <>
      <h1>Welcome to React + vite</h1>
    </>
  );
}

export default App;
