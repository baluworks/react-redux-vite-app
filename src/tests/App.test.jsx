import { render } from '@testing-library/react';
import App from '../App';
test('App Render', () => {
	const { container } = render(<App />);
	expect(container).toMatchSnapshot();
});
