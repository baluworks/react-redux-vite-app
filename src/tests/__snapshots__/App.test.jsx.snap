// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App Render 1`] = `
<div>
  <h1>
    Welcome to React + vite
  </h1>
</div>
`;
